src/utils.ts

```diff
@@ -1,16 +1,20 @@
 import { useEffect, useState } from "react";
-/**
- * Formats a given date object in the format yyyy-mm-dd format
- * @param date 
- * @returns A string of the formatted date
- */
+
+// Formats a given date object in the format yyyy-mm-dd format
 export function formatDate(date: Date) {
   const year = date.getFullYear();
   const month = String(date.getMonth() + 1).padStart(2, "0");
   const day = String(date.getUTCDate()).padStart(2, "0");
   return `${year}-${month}-${day}`;
 }
 
+// Strips time from Date and shifts the day by the given amount
+export const modifyDateDay = (date: Date, dayDiff: number) =>
+  new Date(date.getFullYear(), date.getMonth(), date.getDate() + dayDiff);
+
+// Convert YYYY-MM-DD date string to Date and strip time
+export const dateFromStr = (dateStr: string) => new Date(dateStr + "T00:00:00");
+
```

Comment:
Change these to `export function` for codebase consistency! or change everything else to const if you really want to lol. Consistency is what matter most.